package.json

```diff
@@ -1,10 +1,10 @@
 {
-	"name": "vscode-apache-camel",
-	"displayName": "Language Support for Apache Camel",
-	"description": "Provides completion, validation and documentation features for Apache Camel URI elements in XML DSL.",
-	"license": "Apache-2.0",
-	"version": "0.0.1",
-	"preview": true,
+  "name": "vscode-apache-camel",
+  "displayName": "Language Support for Apache Camel",
+  "description": "Provides completion and documentation features for Apache Camel URI elements in XML DSL.",
+  "license": "Apache-2.0",
+  "version": "0.0.2",
```

Comment:
I already feel the pain if we need to update the version after each commit on master

Reply:
there will be ways around that for sure.